src/test/java/org/wahlzeit_revisited/repository/PhotoRepositoryIT.java

```diff
@@ -0,0 +1,124 @@
+package org.wahlzeit_revisited.repository;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.wahlzeit_revisited.BaseModelTest;
+import org.wahlzeit_revisited.model.*;
+
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+
+public class PhotoRepositoryIT extends BaseModelTest {
+
+    private PhotoRepository repository;
+    private PhotoFactory factory;
+
+    @Before
+    public void setupDependencies() {
+        repository = new PhotoRepository();
+        factory = new PhotoFactory();
+        repository.factory = factory;
+    }
+
+    @Test
+    public void test_insertPhoto() throws SQLException {
+        // prepare
```

Comment:
I think we called these stages ´arrange´, ´act´, ´assert´... Please keep it consistent with that!

Reply:
Changed in: c94bda9a050dc79d85f9364bb8fd4870312edb8e